gluten-core/src/main/resources/substrait/proto/substrait/algebra.proto

```diff
@@ -149,6 +149,28 @@ message ReadRel {
         uint64 max_block_size = 1;
         NamedStruct schema = 2 [deprecated=true];
       }
+      message IcebergReadOptions {
+        enum FileContent {
+          DATA = 0;
+          POSITION_DELETES = 1;
+          EQUALITY_DELETES = 2;
+        }
+        message DeleteFile {
+          FileContent fileContent = 1;
+          string filePath = 2;
+          uint64 fileSize = 5;
```

Comment:
maybe a stupid question: why it skips 3 and 4?

Reply:
I removed some redundant fields and forgot to update the sequence numbers; it shouldn't skip 3 and 4.